src/middleware/login/index.js

```diff
@@ -18,7 +18,7 @@ export const getUser = (payload, dispatch, history) => (
     .then(rejectBadResponse)
     .then(response => response.json())
     .then(user => {
-      ['id_token', 'access_token'].forEach(key => {
+      ['accessToken', 'refreshToken'].forEach(key => {
         localStorage.setItem(key, user[key]);
```

Comment:
I hadn't gotten around to handling an expired token in this PR, but in an upcoming PRs I plan to have middleware that handles certain HTTP statuses, specifically one for auth.

Reply:
We need to absolutely nail down the auth/request middleware that will be talking to the API in order to handle getting an initial access token and refreshing that token when it expires. This needs to happen ASAP, otherwise, there's no way this code is useful at all.  It's untested boilerplate.